velox/exec/tests/SimpleHttpClientTest.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "folly/json.h"
+#include "velox/common/base/VeloxException.h"
+#include "velox/common/base/tests/GTestUtils.h"
+
+#include <map>
+#include <string>
+#include <thread>
+#include "velox/exec/tests/utils/SimpleHttpClient.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec::test;
+
+class SimpleHttpClientTest : public testing::Test {};
+
+TEST_F(SimpleHttpClientTest, get) {
+  auto content = httpGet("https://facebookincubator.github.io/velox/");
+  ASSERT_FALSE(content.empty());
+}
+
+TEST_F(SimpleHttpClientTest, fail) {
+  VELOX_ASSERT_THROW(
+      httpPost("null", "", std::map<std::string, std::string>{}),
+      "Fail to post to null.");
+  VELOX_ASSERT_THROW(httpGet("null"), "Fail to get from null.");
+}
+
+TEST_F(SimpleHttpClientTest, DISABLED_query_hive_runner) {
```

Comment:
Please, add comments to explain why this tests exists, but disabled.

naming: DISABLED_query_hive_runner -> DISABLED_queryPresto

Reply:
Fixed